addon/popup.jsx

```diff
@@ -272,12 +277,28 @@ class MailPreference extends React.Component {
       </footer>
     );
 
+    const providerButtonClasses = showAdditionalProviders ? "provider-button" : "provider-button disabled";
+    const providerCb = showAdditionalProviders ? this.onSelect : () => {};
+
     return <div>
-      <div>
-        <button onClick={this.onSelect.bind(this, "gmail")}>Gmail</button>
-        <button onClick={this.onSelect.bind(this, "yahoo")}>Yahoo</button>
-        <button onClick={this.onSelect.bind(this, "outlook")}>Outlook</button>
+      <p className="provider-heading">Select Default Mail Provider</p>
+      <div className="separator"></div>
+
+      <div className="providers-container">
+        <div className="provider-button" onClick={providerCb.bind(this, "gmail")}>
```

Comment:
These should also be buttons. I think the buttons could just have a `disabled` attribute on them (and the CSS can select on that as well).